Project SF/Views/Tabs/Settings/SettingsView.swift

```diff
@@ -7,6 +7,27 @@
 
 import SwiftUI
 
+struct AboutFooter: View {
+    var body: some View {
+        VStack(spacing: 8) {
+            // I have no idea how to make this centered without this hack.
```

Comment:
@wfltaylor or @Priva28 review this please.

Reply:
Sorry I'm not very good with SwiftUI but otherwise this looks good

Reply:
What do you mean? You want the views inside the VStack to be centered horizontally? In that case add `alignment: .center`. If you want it to be centered vertically add a `Spacer()` at the top and bottom in the VStack